aws_lambda_builders/workflows/dotnet_clipackage/actions.py

```diff
@@ -22,21 +22,22 @@ class GlobalToolInstallAction(BaseAction):
     DESCRIPTION = "Install or update the Amazon.Lambda.Tools .NET Core Global Tool."
     PURPOSE = Purpose.COMPILE_SOURCE
 
-    def __init__(self, subprocess_dotnet):
+    def __init__(self, subprocess_dotnet, tool_dir):
         super(GlobalToolInstallAction, self).__init__()
         self.subprocess_dotnet = subprocess_dotnet
+        self.tool_dir = tool_dir
 
     def execute(self):
         try:
             LOG.debug("Installing Amazon.Lambda.Tools Global Tool")
             self.subprocess_dotnet.run(
-                ['tool', 'install', '-g', 'Amazon.Lambda.Tools'],
+                ['tool', 'install', '--tool-path', self.tool_dir, 'Amazon.Lambda.Tools'],
```

Comment:
This would still be a global tool but with a custom path.
> As a global tool in a custom location (also known as a tool-path tool).
>
>The tool binaries are installed in a location that you specify. You can invoke the tool from the installation directory or by providing the directory with the command name or by adding the directory to the PATH environment variable. One version of a tool is used for all directories on the machine.

Maybe we can use local tools instead and check whether a global Amazon.Lambda.Tools is already installed.
> As a local tool (applies to .NET Core SDK 3.0 and later).
>
>The tool binaries are installed in a default directory. You invoke the tool from the installation directory or any of its subdirectories. Different directories can use different versions of the same tool.

We should also check whether a global version of the Amazon.Lambda.Tools is already installed and skip over if needed.